src/MSONUtility.h

```diff
@@ -366,16 +427,10 @@ namespace mson {
         }
 
         if (typeDefinition.baseType == UndefinedBaseType &&
-            !typeDefinition.typeSpecification.name.symbol.literal.empty()) {
-
-            // WARN: Unable to find the named type in the named type base type table
-            std::stringstream ss;
-            ss << "unable to find the symbol `" << typeDefinition.typeSpecification.name.symbol.literal << "` in the list of named types";
+            !typeDefinition.typeSpecification.name.symbol.literal.empty() &&
+            !typeDefinition.typeSpecification.name.symbol.variable) {
```

Comment:
It's because if the symbol is `*T*`, we don't need to check for the presence of `T`. That is something I have been checking in `addDependency` but forgot to add it here. So, bug fixed.